### PR TITLE
Add getCommunity route to query single community info.

### PR DESCRIPTION
**Pull request by jnaviask**
## Link to Issue
Closes: #8755

## Description of Changes
- Adds GetCommunity TPRC route to get a specific community, as part of the changes required to complete #4976.

## Test Plan
- Will be exercised by a follow-up PR by @mzparacha that will use it to query single community info, rather than querying against the list of all communities in the /communities legacy route.

### CODE REVIEW

**Review by Rotorsoft — APPROVED** [review]
LGTM

File: libs/model/src/community/GetCommunity.query.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+          required: true,
+        });
+      }
+
+      return (
+        await models.Community.findOne({
+          where,
+          include,
+        })
+      )?.toJSON();
```
Comment (rbennettcw) [inline libs/model/src/community/GetCommunity.query.ts:26]: Do we want to throw an error if the community isn't found?

Comment (jnaviask) [reply libs/model/src/community/GetCommunity.query.ts:26]: We should return a null value if not found and let the client handle the exception, unless you see a reason to throw an error in the route itself.

Comment (rbennettcw) [reply libs/model/src/community/GetCommunity.query.ts:26]: I haven't seen us use this convention before, but then again we rarely ever return a single entity. Seems fine to me though– less spam in the logs.

**Review by rbennettcw — APPROVED** [review]

**Pull request merged**